### DISCUSSION TITLE
how to pack two python small projects under a big project

### DISCUSSION THREAD

**Discussion by Yezhj1** [pypa/setuptools#3556] [Q&A]
i have a big project, there are two small projects that i want to create wheel packages. but i don't konw how to handle that problems.

**Comment by Yezhj1** [comment]
please help me

**Comment by abravalheri** [comment]
Hi @Yezhj1, you can keep each project completely separated in 2 folders and distribute one independently of the other (for this you can add a [`pyproject.toml`](https://setuptools.pypa.io/en/latest/userguide/quickstart.html#basic-use) to each's project folder).

Alternatively you can create a single distribution that installs both packages at the same time. This way, you just place a single `pyproject.toml` at the root of your project.
